.shopify-build/polaris-system-integration-test.yml

```diff
@@ -0,0 +1,29 @@
+containers:
+  default:
+    docker: gcr.io/shopify-docker-images/apps/production/web-platform-ci-node:10.18.0-yarn-1.21.1
+  styleguide:
+    docker: gcr.io/shopify-docker-images/apps/production/web-platform-ci-node:10.13.0-yarn-1.21.1
+
+shared:
+  build-polaris: &build-polaris
+    label: 'Build polaris'
+    container: default
+    timeout: 5m
+    dependencies:
+      - yarn
+    run:
+      - ./.shopify-build/build-and-upload-to-gcp.sh
+  build-web: &build-web
+    label: 'Build polaris'
+    trigger: 'web-polaris-builder'
+    build:
+      branch: add-polaris-build-pipeline
+      commit: HEAD
+      env:
+        POLARIS_ARTIFACT_FILENAME: 'shopify-polaris-${BUILDKITE_COMMIT}.tgz'
+
+steps:
+  - <<: *build-polaris
```

Comment:
These steps aren't shared between multiple pipelines, so just put them all inline, to avoid that bit of indirection I reckon ([polaris-styleguide does this](https://github.com/Shopify/polaris-styleguide/blob/master/.shopify-build/polaris-styleguide.yml))

Reply:
I was thinking of reusing these later for tests in other repos like the styleguide but that can come later